test/solidity/oz/transparent-upgradeable-proxy/transparentProxy.js

```diff
@@ -0,0 +1,108 @@
+/*-
+ *
+ * Hedera Smart Contracts
+ *
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+const chai = require('chai')
+const { expect } = require('chai')
+const chaiAsPromised = require("chai-as-promised")
+const { ethers } = require('hardhat')
+const Constants = require('../../../constants')
+const Utils = require('../../../utils')
+chai.use(chaiAsPromised);
+
+describe('@OZ[TransparentUpgradeableProxy] Transparent Upgradeable Proxy', function () {
```

Comment:
Ah I'm sorry this is on me.. It should be something like `@OZTransparentUpgradableProxy` no `[]` (square brackets).

Reply:
Fixed :)